pubspec.lock

```diff
@@ -0,0 +1,551 @@
+# Generated by pub
```

Comment:
I don't think we should have a pubspec file at the top level of the repo.

Reply:
that shows how long I've had the same local client.